exercises/randomgenerator.js

```diff
@@ -4,19 +4,29 @@ class RandomGenerator{
 	constructor(min, max){ 
 		this.min = min;
 		this.max = max;
+		this.randomNumber = null;
+		if (min === 'undefined'){
+			this.min = 1;
```

Comment:
good that you checked for undefined.  checking for falsey would exclude 0.

could also use default params, which you haven't learned yet
``` constructor min=1, max=10) ```